src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -84,6 +83,53 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    //ToDo(kristenmason@): implement the Google Maps request access functionality
+    this.simidProtocol.sendMessage(CreativeMessage.REQUEST_PAUSE).then(() => {
+      findNearest.classList.add("hidden");
+    }).catch(() => {
+      this.simidProtocol.reject(eventData, {errorCode: CreativeErrorCode.PAUSE_NOT_HONORED, 
```

Comment:
What is eventData here?

Reply:
I am restructuring this to sendMessage as opposed to rejecting so I will not be using eventData